ros_gz_sim/launch/ros_gz_sim.launch.py

```diff
@@ -92,20 +103,10 @@ def generate_launch_description():
                           ('config_file', config_file),
                           ('container_name', container_name),
                           ('namespace', namespace),
+                          ('create_own_container', str(False)),
```

Comment:
this is hardcoded, is this right ?

Reply:
That's on purpose, to make sure that only one container is used.